lib/src/presentations/on_boarding/screen/on_boarding_page.dart

```diff
@@ -9,71 +10,75 @@ class OnBoardingPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final width = MediaQuery.sizeOf(context).width;
-
     return Scaffold(
       floatingActionButton: FloatingActionButton(
         onPressed: () {},
         shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(360)),
         elevation: 0,
         child: const Icon(FontAwesomeIcons.arrowRight),
       ),
-      body: SafeArea(
-        child: Padding(
-          padding: const EdgeInsets.all(28),
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              const Align(
-                alignment: Alignment.topRight,
-                child: Text(
-                  "Jobspot",
-                  style: TextStyle(fontWeight: FontWeight.w700, fontSize: 18),
-                ),
-              ),
-              const Spacer(),
-              Align(
-                alignment: Alignment.center,
-                child: SvgPicture.asset(AppImages.onBoarding),
+      body: _body(context),
+    );
+  }
+
+  Widget _body(BuildContext context) {
+    final width = MediaQuery.sizeOf(context).width;
+
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.all(28),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const Align(
+              alignment: Alignment.topRight,
+              child: Text(
+                "Jobspot",
+                style: TextStyle(fontWeight: FontWeight.w700, fontSize: 18),
               ),
-              const Spacer(),
-              RichText(
-                text: TextSpan(
-                  style: const TextStyle(
-                    color: Colors.black,
-                    fontSize: 40,
-                    fontWeight: FontWeight.w700,
-                    height: 0.947,
-                  ),
-                  children: [
-                    const TextSpan(text: "Find Your\n"),
-                    TextSpan(
-                      text: "Dream Job",
-                      style: TextStyle(
-                        color: AppColors.neonCarrot,
-                        decoration: TextDecoration.underline,
-                        decorationColor: AppColors.neonCarrot,
-                      ),
-                    ),
-                    const TextSpan(text: "\nHere!"),
-                  ],
+            ),
+            const Spacer(),
+            Align(
+              alignment: Alignment.center,
+              child: SvgPicture.asset(AppImages.onBoarding),
+            ),
+            const Spacer(),
+            RichText(
+              text: TextSpan(
+                style: const TextStyle(
+                  color: Colors.black,
+                  fontSize: 40,
+                  fontWeight: FontWeight.w700,
+                  height: 0.947,
                 ),
+                children: [
+                  TextSpan(text: "${AppLocalizations.of(context)!.findYour}\n"),
```

Comment:
In this line, Can you find a way to avoid using context in this situation. For example: AppLocal.text.findYour."

Reply:
I fixed it